### PR TITLE
Integrate validation

### PR DESCRIPTION
**Pull request by RabiaSajjad**

### CODE REVIEW

**Comment by wardi** [comment]
this would be simpler if we disable the automatic xloader job creation on resource update and instead solely rely on our fork of ckanext-validation to submit xloader jobs

**Comment by RabiaSajjad** [comment]
> this would be simpler if we disable the automatic xloader job creation on resource update and instead solely rely on our fork of ckanext-validation to submit xloader jobs

Agree, this would cover the manual uploading to the DataStore using the "Upload to DataStore" button on the DataStore tab.

File: ckanext/xloader/plugin.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def is_it_an_xloader_format(cls, format_):
 class xloaderPlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IConfigurer)
     plugins.implements(plugins.IConfigurable)
-    plugins.implements(plugins.IResourceUrlChange)
```
Comment (wardi) [inline ckanext/xloader/plugin.py]: we could check for the validation plugin here and then we don't need the rest of the changes in the PR, right?

File: ckanext/xloader/jobs.py [hunk 2]
```diff
@@ -145,6 +145,30 @@ def xloader_data_into_datastore_(input, job_dict):
...
+            time.sleep(10)
+            logger.info('The resource will be uploaded to datastore after validation is complete')
+
+        try:
+            logger.info('Validation status for the resource is: ' + validation.get('status'))
+            if validation.get('status') != 'success':
+                raise JobError('Only validated resources can be added to the Data Store.')
+        except AttributeError:
+            raise JobError('Unable to validate resource')
+
```
Comment (wardi) [inline ckanext/xloader/jobs.py]: If the file is submitted manually by an admin I think we should still allow it to be loaded, right?

File: ckanext/xloader/plugin.py (hunk 1 above)
Comment (wardi) [reply ckanext/xloader/plugin.py]: or rather, we could keep this implements line and add the check for the plugin in the after_create function below

**Pull request merged**